tools/add_triage_signature.py

```diff
@@ -2399,6 +2399,61 @@ def _process_ticket(self, url, issue_key):
             )
 
 
+class ControllerFailedToStart(ErrorSignature):
+    """
+    This signature looks for missing pivot URL in rpm-ostree status of the control-plane hosts
+    """
+
+    PODS_PATH = "*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/resources/pods.json"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="assisted_controller_failed_to_start",
+            comment_identifying_string="h1. Assisted Installer Controller failed to start",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster_hosts = md["cluster"]["hosts"]
+        bootstrap_node = [host for host in cluster_hosts if host["bootstrap"]][0]
+
+        if bootstrap_node["progress"]["current_stage"] != "Waiting for controller":
+            return
+
+        triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=get_metadata_json(url)["cluster"]["id"])
+        try:
+            # Fetch pods.json from the bootstrap log bundle
+            pods_json = triage_logs_tar.get(self.PODS_PATH)
+        except FileNotFoundError:
+            return
+
+        # Fetch assisted-controller status
+        try:
+            pods = json.loads(pods_json)
+            controller_pod = [pod for pod in pods["items"] if pod["metadata"]["namespace"] == "assisted-installer"][0]
+            ready = [
+                condition["status"] == "True"
+                for condition in controller_pod["status"]["conditions"]
+                if condition["type"] == "Ready"
+            ][0]
+            if not ready:
+                self._update_triaging_ticket(
+                    dedent(
+                        f"""
+                The controller pod isn't ready, here is it's status:
+                h3. conditions:
+                {self._generate_table_for_report(controller_pod["status"]["conditions"])}
+                h3. containerStatuses:
+                {self._generate_table_for_report(controller_pod["status"]["containerStatuses"])}
+                """
+                    )
+                )
+        except Exception:
+            self._update_triaging_ticket("Failed to get controller pod from pods.json")
```

Comment:
```suggestion
        except Exception as e:
            self._update_triaging_ticket("Failed to get readiness of controller pod from installer-gather resources/pods.json: {e}")
```

Include the error in the sig